docs/experiments/request_locking.mdx

```diff
@@ -6,6 +6,20 @@ description: Parallelize crawlers with ease using request locking
 
 import ApiLink from '@site/src/components/ApiLink';
 
+:::tip Release announcement
+
+As of **March 2024** (`crawlee` version `3.9.0`), this experiment is now enabled by default! With that said, if you encounter issues you can:
```

Comment:
if we want to include this in 3.9, we should wait with merging, we might want to ship one more patch release before that. i surely want the tiered proxy PR in there too (and that itself should be a minor bump anyway)

Reply:
I just assumed we'd merge this into 3.9 anyways since its quite a change (but we can always just update docs with correct version)